polaris-react/package.json

```diff
@@ -114,6 +113,7 @@
     "fs-extra": "^7.0.1",
     "glob": "^7.1.4",
     "gray-matter": "^4.0.2",
+    "lodash": "^4.17.21",
```

Comment:
Why do we still need this as a dev dep?

Reply:
We still need it for the dev environment. There are a two functions used in the script which parses the readme examples into stories.

Reply:
Here https://github.com/Shopify/polaris/blob/2141ca5f31526d50421011841d389800be05a2af/polaris-react/.storybook/polaris-readme-loader.js#L411